MVP.md

```diff
@@ -11,84 +11,12 @@ even on mobile devices, which leads to roughly the same functionality as
 This document explains the contents of the MVP at a high-level. There are also
 separate docs with more precise descriptions of:
 
+ * [Modules](Modules.md)
  * [Polyfill to JavaScript](Polyfill.md);
  * [AST semantics](AstSemantics.md);
  * [Binary encoding](BinaryEncoding.md);
  * Implementation [in the browser](Web.md) and [outside the browser](NonWeb.md).
 
-**Note**: This content is still in flux and open for discussion.
```

Comment:
I'm happy to, I was just thinking that this was originally added when V1.md had a bunch of text and so now it seems a bit ad hoc and asymmetric since most files don't have it.  The important thing is the Readme.md has it bold.  Still want to keep?